composer.json

```diff
@@ -3,6 +3,12 @@
     "description": "An automated build tool to allow projects to have a set standardized operations scripts.",
     "license": "GPL-2.0-or-later",
     "type": "composer-plugin",
+    "repositories": [
+        {
+            "type": "git",
+            "url": "https://github.com/Lullabot/composer-checks.git"
+        }
+    ],
```

Comment:
As you see from the above CI checks, `repositories` is a root-only value. Instead, let's publish the plugin to packagist like a normal plugin.

https://github.com/Lullabot/composer-checks/issues/3